sketcherMinimizerAtom.cpp

```diff
@@ -389,7 +389,7 @@ void sketcherMinimizerAtom::writeStereoChemistry() // sets stereochemistry for
         return;
     }
     size_t n = neighbors.size();
-    if (n != 3 && n != 4) {
+    if (!canBeChiral() || (n != 3 && n != 4)) {
```

Comment:
Wait - does this mean that the carbon atoms in the JIRA case aren't going to be marked as chiral at all? That seems wrong, too.

Reply:
well they are, and it makes sense cause they have no Hs. We have another case to track that problem so in the end they won't, but as far as this case is concerned I think this is the appropriate behaviour for coordgen

Reply:
Sorry, I'm not sure I understand. The C atoms may not have any explicit H atoms, but implicit ones should also be considered, shouldn't they?

Reply:
So, I guess when I see this I have a couple of questions:

1. What is `(n != 3 && n != 4)` even doing? Isn't that logic already performed in `CanBeChiral()`?
2. Should there be some earlier check or assertion that we're not setting stereochemistry on something that can't even be chiral in the first place?
3. What about trigonal nitrogens? Do you allow them to be have parity?

Reply:
@ricrogz the problem is that we were marking the atoms to have 0 implicit Hs. So when I have no Hs I mean no implicit and no explicit Hs.

Reply:
@d-b-w 1) you are right
2) this looks like the right place to check to me. It's at the beginning of where we write stereochemistry to the atom
3) yes